cmd/keymasterd/main.go

```diff
@@ -1973,6 +1978,7 @@ func (state *RuntimeState) u2fTokenManagerHandler(w http.ResponseWriter, r *http
 	returnAcceptType := getPreferredAcceptType(r)
 	switch returnAcceptType {
 	case "text/html":
+		eventNotifier.PublishWebLoginEvent(authUser)
```

Comment:
There is no weblogin here.. remove this

Reply:
Where else am I supposed to get a web login event when U2F is required?